ledger/src/leader_schedule.rs

```diff
@@ -2,7 +2,7 @@ use {
     itertools::Itertools,
     rand::distributions::{Distribution, WeightedIndex},
     rand_chacha::{rand_core::SeedableRng, ChaChaRng},
-    solana_sdk::pubkey::Pubkey,
+    solana_pubkey::Pubkey,
```

Comment:
I think we can make this substitution on a bunch more files in `solana_ledger`. Looks like:
- `benches/blockstore.rs`
- `src/blockstore.rs`
    - This file looks to have a redundant import in the test module as well
- `src/blockstore_db.rs`
- `src/blockstore_processor.rs`
    - Redundant import in test module as well
- `src/genesis_utils.rs`
- `src/leader_schedule.rs`
- `src/leader_schedule_cache.rs`
- `src/leader_schedule_utils.rs`
- `src/shred.rs`
- `src/shredder.rs`
- `src/sigverify_shreds.rs`
- `src/staking_utils.rs`
- `src/token_balances.rs`
- `src/transaction_address_lookup_table_scanner.rs`
- `src/shred/merkle.rs`

Reply:
let's do this in a separate PR?

Reply:
I'm somewhat inclined to include it in this same PR. The PR title states "replace solana_sdk::pubkey with solana_pubkey in ledger", and without these changes, that is not fully accomplished. The diff should still be pretty small with these changes. Finally, if we were to do those changes in a separate PR, then I'd argue we should revert this line in `src/leader_schedule.rs` for the sake of consistency across the crate.

That being said, I don't want to punish a good deed. So, if you're opposed to tagging the change onto this PR, I'd probably still be fine pushing the PR.

Reply:
ah by that PR title I meant `sed -i 's/solana_sdk::pubkey/solana_pubkey/g'`